test/test_recog.py

```diff
@@ -82,140 +77,6 @@ def init_chainer_weight_const(m, val):
         p.data[:] = val
 
 
-@pytest.mark.execution_timeout(5)
-@pytest.mark.skipif(is_torch_1_2_plus, reason="pytestskip")
```

Comment:
Do you know why these tests are skipped when the torch version >= 1.2?

Reply:
According to #1840, we gave up on this compatibility.

Reply:
OK!